astrowidgets/tests/widget_api_test.py

```diff
@@ -138,12 +144,13 @@ def test_marking_operations(self):
         assert not self.image.click_drag
         assert not self.image.scroll_pan
 
-        # Regression test for GitHub Issue 97:
-        # Marker name with no markers should give warning.
-        with pytest.warns(UserWarning, match='is empty') as warning_lines:
+        # Make sure no warning is issued when trying to retrieve markers
+        # with a name that does not exist.
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")
```

Comment:
Hmm, won't setting pytest to turn all warnings into exceptions take care of this?